src/components/content/textStyle.js

```diff
@@ -0,0 +1,66 @@
+const TextContentVariant = {
+    short: 'short',
+    paragraph: 'paragraph',
+}
+
+const TextContent = {
+    [TextContentVariant.short]: {
```

Comment:
define when use the "short" key in application. Is this the same when apply in any screens, any components